provided.al2/graalvm/java11/cookiecutter-aws-sam-graalvm-gradle/{{cookiecutter.project_name}}/Dockerfile

```diff
@@ -1,5 +1,12 @@
+ARG PLATFORMARG=linux/amd64
+        ### Example: "linux/amd64", "windows/amd64", "linux/aarch64", "darwin/aarch64", ..  See supported values at: https://github.com/graalvm/graalvm-ce-builds/releases/
+### !! Attention !! Repeat this for the "ENV" line below for "PLATFORM" env-variable.
```

Comment:
Can we re-format comment lines?

Reply:
Absolutely.  Please help get the advanced-user's attention to the 2 places that changes need to made (to keep them in sync).